homeassistant/components/fitbit/sensor.py

```diff
@@ -153,7 +179,7 @@ class FitbitSensorEntityDescription(SensorEntityDescription):
     ),
     FitbitSensorEntityDescription(
         key="activities/distance",
-        name="Distance",
+        translation_key="distance",
```

Comment:
This probably doesn't need a translation key as it can use the device class translation